build-logic/src/main/groovy/qsl.module.gradle

```
@@ -192,6 +192,7 @@ javadoc {
 license {
 	rule rootProject.file("codeformat/COLONEL_MODIFIED_HEADER")
 	rule rootProject.file("codeformat/FABRIC_MODIFIED_HEADER")
+	rule rootProject.file("codeformat/FOUNDATION_GAMES_MODIFIED_HEADER")
```

My one cent on this (the rest of the review will come later):
```suggestion
	rule rootProject.file("codeformat/FOUNDATIONGAMES_MODIFIED_HEADER")
```
it's so weird to see the underscore there, it makes FoundationGames look like a company instead of a person :P